app/views/variable_generation_processes/edit.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="row row-content">
+  <section>
+    <h2>Variable Generation for <%= current_user.provider_id %></h2>
+
+    <fieldset>
+      <h4>Augmentation</h4>
+      <!-- figure out a box -->
+      <%= form_tag(variable_generation_process_path, method: :put, class: 'something') %>
```

Comment:
something?

Reply:
hah. i'll remove this. It doesn't actually need a class, but I thought not having one was causing an error.